geps/gep-1713/index.md

```diff
@@ -164,15 +164,18 @@ type ListenerEntry struct {
 	// Port is the network port. Multiple listeners may use the
 	// same port, subject to the Listener compatibility rules.
 	//
-	// If the port is specified as zero, the implementation will assign
+	// If the port is not set or specified as zero, the implementation will assign
 	// a unique port. If the implementation does not support dynamic port
 	// assignment, it MUST set `Accepted` condition to `False` with the
 	// `UnsupportedPort` reason.
 	//
 	// Support: Core
 	//
 	// +optional
-	Port *PortNumber `json:"port,omitempty"`
+	//
+	// +kubebuilder:validation:Minimum=0
```

Comment:
Do you mind leaving this change out until we have docs for ListenerSet? At this point these are the only docs we have for ListenerSet, so I'd rather this match what's been released in the API until we can add those. (https://github.com/kubernetes-sigs/gateway-api/blob/b9062b78fa40afb27d0701ef61b4db7756241c81/apisx/v1alpha1/xlistenerset_types.go#L130)